stable/yugabyte/templates/setup-credentials-configmap.yaml

```
@@ -37,12 +37,12 @@ data:
     {{- end }}
 
     prefix_ysql_cmd=(
-      /home/yugabyte/bin/ysqlsh -h yb-tservers.{{ .Release.Namespace }}.svc.{{ .Values.domainName }}
+      /home/yugabyte/bin/ysqlsh -h {{ .Release.Name }}-yb-tservers.{{ .Release.Namespace }}.svc.{{ .Values.domainName }}
```

```suggestion
      /home/yugabyte/bin/ysqlsh -h {{ $tserverFQDN }}
```
We can do the same thing for all the occurrences.